test/Controller/ArticleControllerTest.php

```diff
@@ -1225,6 +1225,7 @@ public function it_displays_example_rds_info_bar_when_it_has_an_example_rds()
             'This research is available in a reproducible view.',
             array_map('trim', $crawler->filter('.info-bar--warning')->extract(['_text']))
         );
+        $this->assertEmpty($crawler->filter('.article-download-links-list__link')->selectLink('Executable DAR'));
```

Comment:
selecting by text is a bit fragile, I guess there's no class or ID?

Reply:
`a[data-download-type="dar-download"]` is the other option. But I think text is ok here.

Reply:
it matches other assertions, though the ones I've seen are more comparing expected and actual text, so easy to update in case of failure

Reply:
(minor point)